packages/services/src/view-service/unbonding-tokens-by-address-index/index.ts

```diff
@@ -31,9 +44,29 @@ export const unbondingTokensByAddressIndex: Impl['unbondingTokensByAddressIndex'
         continue;
       }
 
+      const regexResult = assetPatterns.unbondingToken.capture(
+        getDisplayFromBalancesResponse(new BalancesResponse(balancesResponse)) ?? '',
+      );
+      if (!regexResult) throw new Error('expected delegation token identity key not present');
+
+      const validatorInfoResponse = await stakingClient.getValidatorInfo({
+        identityKey: identityKeyFromBech32m(regexResult.idKey),
+      });
+      const validatorInfo = getValidatorInfo(validatorInfoResponse);
+      const extendedMetadata = new Any({
+        typeUrl: ValidatorInfo.typeName,
+        value: validatorInfo.toBinary(),
+      });
+
+      const withValidatorInfo = getBalanceView(new BalancesResponse(balancesResponse)).clone();
+      if (withValidatorInfo.valueView.case !== 'knownAssetId')
+        throw new Error(`Unexpected ValueView case: ${withValidatorInfo.valueView.case}`);
+
+      withValidatorInfo.valueView.value.extendedMetadata = extendedMetadata;
```

Comment:
Same way as for delegation tokens 
https://github.com/penumbra-zone/web/blob/b97e2102767369a835e4fc6b1df49ce0ab30f9f3/packages/services/src/view-service/delegations-by-address-index.ts#L77-L84